src/unity/lib/unity_sframe.cpp

```diff
@@ -1950,6 +1950,188 @@ void unity_sframe::explore(const std::string& path_to_client, const std::string&
       }
     };
 
+    auto getAccordian = [self, &reader, &ew, &column_names, &empty_tz](std::string column_name, size_t index) {
+        index = index - 1;
+
+        ASSERT_TRUE(std::find(column_names.begin(), column_names.end(), column_name) != column_names.end());
+        DASSERT_LT(index, self->size());
+        DASSERT_GE(index, 0);
+
+        auto modal_sa = self->select_column(column_name);
```

Comment:
What is `modal`? Should this be `accordion`?

Probably should combine this line with the one below, if we only need the gl_sarray after this line.

Reply:
alright